src/framework/mlt_types.h

```diff
@@ -148,7 +148,9 @@ typedef enum
 	filter_type,                /**< Filter class */
 	transition_type,            /**< Transition class */
 	consumer_type,              /**< Consumer class */
-	field_type                  /**< Field class */
+	field_type,                 /**< Field class */
+	link_type,                  /**< Link class */
+	chain_type                  /**< Chain class */
```

Comment:
Since we are breaking API, I want to change all these enum symbols to start like `mlt_type_...`

Reply:
done